packageregistry/ruby.go

```diff
@@ -136,6 +140,10 @@ func (np *rubyPackageDiscovery) GetPackage(packageName string) (*Package, error)
 	return convertGemObjectToPackage(gemObject)
 }
 
+func (np *rubyPackageDiscovery) GetPackageDownloadStats(packageName string) (DownloadStats, error) {
+	return DownloadStats{}, nil
```

Comment:
Return error please ie. download stats is not supported, otherwise it will cause bugs in client code assuming it to be `0` downloads